README.md

```diff
@@ -258,6 +273,7 @@ Arguments:
 - name: The name of the publication
 - collection: A function that returns the collection to publish (or just a collection, if it's insecure).
 - selector: (Optional) A function that returns mongo selector that will limit the results published (or just the selector).
+- settings: (Optional) A object with settings on server side's publish function. (Details see below)
```

Comment:
Ok, I think you're right about needing a separate server-side option. The client-side option doesn't need to passed to the server, but I do think it needs to exist for people using the package client-side only, since this won't be backwards-compatible for anyone who was using regexes before.